WordPress/Classes/Networking/AccountSettingsRemote.swift

```diff
@@ -16,27 +15,26 @@ class AccountSettingsRemote: ServiceRemoteREST {
         //
         // However it might be possible that the account gets deallocated and
         // when it's fetched again it would create a different api object.
-        let key = api.authToken.hashValue
         // FIXME: not thread safe
         // @koke 2016-01-21
-        if let remote = remotes.objectForKey(key) {
+        if let remote = remotes.objectForKey(api) {
             return remote as! AccountSettingsRemote
         } else {
-            let remote = AccountSettingsRemote(api: api)
-            remotes.setObject(remote, forKey: key)
+            let remote = AccountSettingsRemote(wordPressComRestApi: api)
```

Comment:
I caught this a bit late, but I think the initializer argument label shouldn't have changed here. In any case, make sure to change it back to `api:` when the migration is complete.